util/chplenv/utils.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
That sounds reasonable. I was just staying consistent. The same could be said for the following files:

* chpl_3p_dlmalloc_configs.py
* chpl_3p_gmp_configs.py
* chpl_3p_hwloc_configs.py
* chpl_3p_massivethreads_configs.py
* chpl_3p_qthreads_configs.py
* chpl_3p_re2_configs.py
* chpl_3p_tcmalloc_configs.py

I'm OK with removing the shebang / executable status from all of these including utils.py.